src/axom/inlet/inlet_utils.cpp

```diff
@@ -165,5 +192,28 @@ void markAsStructCollection(axom::sidre::Group& target)
   }
 }
 
+void markRetrievalStatus(axom::sidre::Group& target, const ReaderResult result)
+{
+  target.createViewScalar("retrieval_status", static_cast<int>(result));

```

Comment:
Not sure if this is the right place to do this, but would it make sense to add an entry for "USING_DEFAULT" when the user didn't supply an entry, so inlet uses the default value.

(this might be a separate annotation though, and/or might already be supported in another way)

Reply:
I hadn't considered this until you mentioned it, but I think the information we're adding to the datastore here is sufficient to determine if the default value was used (which was impossible in the general case before this PR).  

I think if we wanted to later report this information (for documentation purposes, maybe), we could just check if the retrieval status was `Success` - if not, the default would have been used.